### PR TITLE
created missing telnyx explanation

### PR DESCRIPTION
**Pull request by yumminova**

### CODE REVIEW

File: core/kazoo_numbers/doc/knm_telnyx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+## Documentation
+* [Telnyx general documentation](https://developers.telnyx.com/docs/api/v1/numbers)
+
+
+## system_config
+#### Configuring number manager
+
+To enable kazoo number manager to use the telnyx module you will need to first update the `number_manager` document in the `system_config` database.
+
+In the configuration section that applies to your crossbar servers (or the default) add `"knm_voxbone"` to the `carrier_modules` parameter.
```
Comment (sergey-safarov) [inline core/kazoo_numbers/doc/knm_telnyx]: Why Voxbone reference present in knm_telnyx documentation?

Comment (yumminova) [reply core/kazoo_numbers/doc/knm_telnyx]: Good question, let me remove that

Comment (yumminova) [reply core/kazoo_numbers/doc/knm_telnyx]: i investigated the situation, and i have come to the conclusion that it must have been a typo.

File: core/kazoo_numbers/doc/knm_telnyx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+### Configuring the telnyx module
+```json
+{
+   "_id": "number_manager.telnyx",
+   "_rev": "XXXXXXXXXXXXX",
+   "default": {
+       "debug": "true",       
+       "enable_provisioning": "true",
+       "monthly_recurring_cost": "1",
```
Comment (sergey-safarov) [inline core/kazoo_numbers/doc/knm_telnyx]: no exist on telnyx module

**Comment by yumminova** [comment]
i received en email with commment: no exist on telnyx module

But i dont know what u mean, there are no docs or examples, and this works.
So if u know how that doc should look, feel free to change it or show me the way

**Comment by yumminova** [comment]
https://github.com/2600hz/kazoo-crossbar/blob/master/priv/couchdb/schemas/system_config.number_manager.telnyx.json

Thats where i got the fields, let me know if that is not correct

**Comment by sergey-safarov** [comment]
yes, I wrote false comment and then deleted.
For me PR looks fine

**Comment by yumminova** [comment]
Ok, that why I didn't see it)) I'm not a github user so was confused.

Thanks

**Comment by jamesaimonetti** [comment]
@yumminova big thanks again! I cleaned up the file a bit and added it to the documentation index. Should be live at https://docs.2600hz.com/dev/core/kazoo_numbers/doc/knm_telnyx/

**Comment by yumminova** [comment]
I'm a captain, so thanks for cleaning up a little))
I barely know how to do it, but improving 😊

Verzonden met ProtonMail Mobile

-------- Oorspronkelijk bericht --------
Aan 8 jan. 2020 01:44, James Aimonetti wrote:

**Pull request closed without merging**